### PR TITLE
fix a bunch of Invalid Escape warnings

### PR DESCRIPTION
**Pull request by jbrockmendel**

### CODE REVIEW

**Comment by josef-pkt** [comment]
#2868 added a unit test for the table latex replacement characters, so this change should be fine test runs are green (or as green as they get given problems in unrelated places)

**Comment by bashtage** [comment]
@josef-pkt LGTM

**Comment by jbrockmendel** [comment]
False-positive errors are a higher priority.  Closing.

File: statsmodels/robust/norms.py [hunk 1]
```diff
@@ -233,7 +233,7 @@ def psi(self, z):
         return test * z + (1 - test) * self.t * np.sign(z)
 
     def weights(self, z):
-        """
+        r"""
```
Comment (bashtage) [inline statsmodels/robust/norms.py]: I think you need to remove the escapes below now that you added r.  \| will render \| not |.

File: statsmodels/robust/norms.py [hunk 2]
```diff
@@ -211,7 +211,7 @@ def rho(self, z):
                 (1 - test) * (np.fabs(z) * self.t - 0.5 * self.t**2))
 
     def psi(self, z):
-        """
+        r"""
```
Comment (bashtage) [inline statsmodels/robust/norms.py]: See http://www.statsmodels.org/stable/generated/statsmodels.robust.norms.HuberT.weights.html#statsmodels.robust.norms.HuberT.weights .  The old text renders correctly.  Usually only need r""" for latex.

Comment (jbrockmendel) [reply statsmodels/robust/norms.py]: > The old text renders correctly. Usually only need r""" for latex

We'd have to ask past-me to be sure, but I think there were a bunch of warnings in the test output about invalid escapes.  These particular locations for r's would have been based on those warnings.

Comment (bashtage) [reply statsmodels/robust/norms.py]: Which format were the warnings for?  I don't think they are there for HTML.  Possibly LaTeX?

**Comment by bashtage** [comment]
Looks like I was wrong about the escapes.  Looks like these were needed.  Sorry for the trouble.

```
/home/travis/build/statsmodels/statsmodels/statsmodels/robust/norms.py:docstring of statsmodels.robust.norms.AndrewWave.psi:8: WARNING: Undefined substitution referenced: "z".
...
/home/travis/build/statsmodels/statsmodels/statsmodels/robust/norms.py:docstring of statsmodels.robust.norms.Hampel.weights:12: WARNING: Undefined substitution referenced: "z|*(c-b)) for b < |z".
```

**Comment by jbrockmendel** [comment]
I've lost track: does this need edits?

**Comment by bashtage** [comment]
Yes, I think you need to revert the last commit.

**Pull request closed without merging**